addons/mail/models/discuss/ir_websocket.py

```diff
@@ -13,12 +13,12 @@ def _get_im_status(self, data):
         im_status = super()._get_im_status(data)
         if "mail.guest" in data:
             # sudo: mail.guest - necessary to read im_status from other guests, information is not considered sensitive
-            im_status["Persona"] += [{**g, 'type': "guest"} for g in (
-                self.env["mail.guest"]
-                .sudo()
-                .with_context(active_test=False)
-                .search_read([("id", "in", data["mail.guest"])], ["im_status"])
-            )]
+            guests = self.env["mail.guest"].sudo().with_context(active_test=False).search_read([("id", "in", data["mail.guest"])], ["im_status"])
```

Comment:
as there are very few values (just `guest_id` and `im_status` it would be better to just replace `**g` and `type` by listing the 2 keys we want in the original code rather than rewriting it like this